presto-spi/src/main/java/io/prestosql/spi/type/Timestamps.java

```diff
@@ -68,7 +68,7 @@
 
     private Timestamps() {}
 
-    static long round(long value, int magnitude)
+    public static long round(long value, int magnitude)
```

Comment:
This should go before `Support for variable precision timestamps in Hive connector` commit (it's currently after).
Or just squash.